### PR TITLE
add_some_plugin

### PR DESCRIPTION
**Pull request by ChenChihChiang**
add_some_plugin
use newer version tag "lts"
add jenkins slave idle time

### CODE REVIEW

File: 02-install-istio-jenkins/jenkins/jenkins-values.yaml [hunk 1]
```diff
@@ -224,6 +224,13 @@ master:
     - gitlab-plugin:1.5.13
     - pipeline-utility-steps:2.5.0
     - permissive-script-security:0.6
+    - blueocean:1.22.0
+    - slack:2.40
+    - role-strategy:2.16
+    - pipeline-github-lib:1.0
+    - pipeline-utility-steps:2.5.0
+    - aws-secrets-manager-credentials-provider:0.2.2
+    - gitlab-plugin:1.5.13
```
Comment (RicoToothless) [inline 02-install-istio-jenkins/jenkins/jenkins-values.yaml]: gitlab-plugin duplicate with line 224
https://github.com/RicoToothless/aws-cdk-jenkins-workshop/pull/4/files#diff-889a35bc07bdcb2f51dc451f024d079fR224

File: 02-install-istio-jenkins/jenkins/jenkins-values.yaml [hunk 2]
```diff
@@ -66,7 +66,7 @@ master:
   # Used for label app.kubernetes.io/component
   componentName: "jenkins-master"
   image: "jenkins/jenkins"
-  tag: "2.204.4"
+  tag: "lts"
```
Comment (RicoToothless) [inline 02-install-istio-jenkins/jenkins/jenkins-values.yaml]: we should fixed jenkins version

**Review by RicoToothless — CHANGES_REQUESTED** [review]
1. fixed jenkins version
2. plugin duplicate

**Comment by ChenChihChiang** [comment]
done

**Pull request merged**